plugins/newsql/newsql.c

```diff
@@ -2191,8 +2203,10 @@ static int handle_newsql_request(comdb2_appsock_arg_t *arg)
 
     CDB2SQLQUERY *sql_query = query->sqlquery;
 
-    if (!clnt.admin && do_query_on_master_check(dbenv, &clnt, sql_query))
+    if (!clnt.admin && do_query_on_master_check(dbenv, &clnt, sql_query)) {
+        logmsg(LOGMSG_ERROR, "query on master, will be rejected\n");
```

Comment:
This will cause log spewing.

Reply:
This PR is to help me track an issue with a specific db. I am not planning to merge it into the trunk.

Reply:
But, if you think these are useful enough, I can add them with lower log level (debug).